internal/service/service_test.go

```diff
@@ -67,3 +67,15 @@ func TestDepGraphToSBOM(t *testing.T) {
 		})
 	}
 }
+
+func TestValidateSBOMFormat_EmptyFormat(t *testing.T) {
+	err := ValidateSBOMFormat("")
+	assert.ErrorContains(t, err, "Must set `--format` flag to specify an SBOM format. "+
+		"Available formats are: cyclonedx1.4+json, cyclonedx1.4+xml, spdx2.3+json")
+}
+
+func TestValidateSBOMFormat_InvalidFormat(t *testing.T) {
```

Comment:
Should we add the case when it is valid and returns nil?

Reply:
I don’t think that's necessary, since we already have a success-path test case prior to this one which covers this case.